### ISSUE TITLE
Renaming files

### ISSUE THREAD

**Issue by dangelion** [dsktschy/laravel-mix-copy-watched#8]
Hi, when I rename a file your plugin doesn't rename the file in the `/dist` but it deletes the file (and doesn't add the file with new name).

**Steps to reproduce**
I add this file:
`resources/images/hello-world.jpg`
and the plugin adds it to
`public/dist/images/hello-world.jpg`

Now I rename to
`resources/images/hello.jpg`
and the plugin removes `public/dist/images/hello-world.jpg` and doesn't add `public/dist/images/hello.jpg`

My `webpack.mix.js`:
```
const mix = require('laravel-mix');
require('laravel-mix-copy-watched');

mix
  .js('resources/js/app.js', 'dist/js')
  .sass('resources/scss/app.scss', 'dist/css')
  .copyDirectoryWatched('resources/fonts', 'public/dist/fonts', { base: 'resources/fonts' })
  .copyDirectoryWatched('resources/images', 'public/dist/images', { base: 'resources/images' })
  .copyDirectoryWatched('resources/videos', 'public/dist/videos', { base: 'resources/videos' })
  .disableSuccessNotifications()
  .browserSync('localhost:8000');
```

Also, I noticed the plugin doesn't work when I add new images. It seems there's yet a lot to work to bring this in production.

**Comment by dsktschy** [comment]
@dangelion 
Thank you for the report.
I tried it, but could not reproduce it.

![issue-8](https://user-images.githubusercontent.com/14947083/64076250-697b6f00-ccfd-11e9-90a0-c9cd420ca9f3.gif)

Can you tell me the followings?
* OS
* Version of Node.js
* Version of laravel-mix version
* Version of laravel-mix-copy-watched
* Command which you executed

**Comment by dsktschy** [comment]
@dangelion
I could reproduce this problem with laravel-mix-copy-watched@2.2.0, and fixed it in v2.2.1.
Can you update laravel-mix-copy-watched to v2.2.1 and retry?

**Comment by dsktschy** [comment]
This problem should have been resolved in v2.2.2.1, so closed.
If this still confirms the problem, please let me know.
